src/api/app/models/project.rb

```diff
@@ -1675,6 +1675,8 @@ def has_local_distribution(project_name, repository)
   end
 
   def status_reports(checkables)
+    checkables = checkables.reject { |checkable| checkable.required_checks.empty? }
```

Comment:
if we are speaking about performance, maybe this should be do in the SQL level :wink:

Reply:
The perf problem is the backend call.

Reply:
and in SQL the required_checks does not even exist - as it's some JSON dump :(

Not sure how active record would handle that. But storing the required checks somewhere else is on the midterm TODO

Reply:
Actually I tried it and it works

```
Repository.where.not(required_checks: nil)
```

so changed...